CMakeLists.txt

```diff
@@ -256,7 +256,7 @@ else()
 	#message(STATUS "catkin DISABLED")
 endif()
 
-find_package(PythonInterp 2.7 REQUIRED)
+find_package(PythonInterp REQUIRED)
```

Comment:
That is from master
See https://github.com/PX4/Firmware/commit/7a698952fbe386d190a40cf3ba68fba18d1fbfa6